controllers/config.go

```diff
@@ -102,6 +112,54 @@ func InitFromFlags(fs *flag.FlagSet, cfg *ManagerConfig) {
 
 	cfg.SecretControllerConfig = &secret.Config{}
 	secret.InitFromFlags(fs, cfg.SecretControllerConfig)
+
+	return nil
+}
+
+// InitFeatureGates initializes feature gates from the provided CLI flag set.
+func InitFeatureGates(fs *flag.FlagSet, featureFlags *featuregate.MutableFeatureGate) error {
```

Comment:
```suggestion
func initFeatureGates(fs *flag.FlagSet, featureFlags *featuregate.MutableFeatureGate) error {
```

Since this is not being used in any other package, please make it a private function.